recipes/jmx_exporter.rb

```diff
@@ -1,37 +1,34 @@
 include_recipe 'optoro_consul::client'
 
-package 'jmx_prometheus_httpserver' do
-  action :upgrade
-end
-
 cookbook_file '/etc/jmx_exporter/jmx_exporter.yaml' do
   source 'jmx_exporter.yaml'
   owner 'root'
   group 'root'
   mode '0644'
 end
 
+service 'jmx_exporter' do
+  action [:disable, :stop]
+end
+
 cookbook_file '/etc/init/jmx_exporter.conf' do
+  action :delete
   source 'jmx_exporter.init'
   owner 'root'
   group 'root'
   mode '0755'
 end
 
-service 'jmx_exporter' do
-  action [:enable, :start]
-end
-
 consul_definition 'kafka-metrics' do
   type 'service'
   parameters(
-    port: 9200,
+    port: 7071,
```

Comment:
9200 also conflicts with ES. So may not want to use it. But also not a problem most likely.